src/dags/bouwstroompunten.py

```diff
@@ -1,23 +1,26 @@
 import json
 import operator
+from typing import Any
 
+import dsnparse
 import requests
 from airflow import DAG
 from airflow.models import Variable
-from airflow.operators.bash import BashOperator
-from airflow.operators.python import PythonOperator
-from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.postgres_operator import PostgresOperator
```

Comment:
The original imports were good. This changes it to deprecated ones.
You should use:
```
from airflow.operators.python import PythonOperator
from airflow.providers.postgres.operators.postgres import PostgresOperator
```

Reply:
:+1: